x-pack/plugins/secops/public/store/local/hosts/selectors.ts

```diff
@@ -4,29 +4,69 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { fromKueryExpression } from '@kbn/es-query';
+import { get } from 'lodash/fp';
 import { createSelector } from 'reselect';
 
 import { State } from '../../reducer';
-import { HostsModel } from './model';
+import { GenericHostsModel, HostsType } from './model';
 
-const hostsQuery = (state: State): HostsModel => state.local.hosts;
+const selectHosts = (state: State, hostsType: HostsType): GenericHostsModel =>
+  get(hostsType, state.local.hosts);
 
-export const authenticationsSelector = createSelector(
-  hostsQuery,
-  hosts => hosts.query.authentications
-);
+export const authenticationsSelector = () =>
```

Comment:
Why were these turned into functions? Was something wrong or is there something I'm missing?